lib/node_modules/@stdlib/blas/base/drot/README.md

```diff
@@ -0,0 +1,216 @@
+var y0 = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+// Create offset views...
+var x1 = new Float64Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
+var y1 = new Float64Array( y0.buffer, y0.BYTES_PER_ELEMENT*3 ); // start at 4th element
+
+drot( 3, x1, -2, y1, 1, 0.707, 0.707 );
+// x0 => <Float64Array>[ 1.0, 9.898, 3.0, 10.604999999999999, 5.0, 11.312 ]
+// y0 => <Float64Array>[ 7.0, 8.0, 9.0, 2.8279999999999994, 4.949, 7.07 ]
+```
+
+#### drot.ndarray( N, x, strideX, offsetX, y, strideY, offsetY, c, s )
+
+Applies a plane rotation using alternative indexing semantics.
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+
+drot.ndarray( x.length, x, 1, 0, y, 1, 0, 1.0, 0.0 );
+// x => <Float64Array>[ 1.0, 2.0, 3.0, 4.0, 5.0 ]
+// y => <Float64Array>[ 6.0, 7.0, 8.0, 9.0, 10.0 ]
+```
+
+The function has the following additional parameters:
+
+-   **offsetX**: starting index for `x`.
+-   **offsetY**: starting index for `y`.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on starting indices. For example, to apply rotation to every other value in `x` starting from second value,
+
+```javascript
+var Float64Array = require( '@stdlib/array/float64' );
+
+var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var y = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+
+drot.ndarray( 4, x, 0, 2, y, 0, 2, 0.707, 0.707 );
+// x => <Float64Array>[ 1.0, 2.0, -2.998188273612, 4.0, 5.0, 6.0 ]
+// y => <Float64Array>[ 7.0, 8.0, -8.994564820835999, 10.0, 11.0, 12.0 ]
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   If `N <= 0`, both functions leave `x` and `y` unchanged.
+-   `drot()` corresponds to the [BLAS][blas] level 1 function [`drot`][drot].
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var drot = require( '@stdlib/blas/base/drot' );
+
+var opts = {
+    'dtype': 'float64'
+};
+var x = discreteUniform( 10, 0, 500, opts );
+console.log( x );
+
+var y = discreteUniform( x.length, 0, 255, opts );
+console.log( y );
+
+// Apply a plane rotation :
+drot( x.length, x, 1, y, 1, 0.707, 0.707 );
+console.log( x );
+console.log( y );
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/blas/drot`][@stdlib/blas/drot]</span><span class="delimiter">: </span><span class="description">applies plane rotation.</span>
```

Comment:
```suggestion
```

As mentioned in the comment above, don't manually edit this section. For new packages, remove the contents entirely.